assets/src/dashboard/components/tooltip/index.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import propTypes from 'prop-types';
+import styled from 'styled-components';
+import { useState } from 'react';
+
+export const Content = styled.div`
+  visibility: ${({ visible }) => (visible ? 'visible' : 'hidden')};
+  width: max-content;
+  position: absolute;
+  right: 0;
+  border-radius: 2px;
+  padding: 10px;
+  background: ${({ theme }) => theme.tooltip.background};
+  color: ${({ theme }) => theme.tooltip.color};
+  opacity: ${({ visible }) => (visible ? 1 : 0)};
+  transition: opacity linear 200ms;
```

Comment:
nit: transition shorthand should be `name | duration | timing function` 
<img width="334" alt="Screen Shot 2020-05-15 at 12 31 28 PM" src="https://user-images.githubusercontent.com/10720454/82089268-2b19ba80-96a8-11ea-9683-2deec037d261.png">